test/conformance/serial/ptp.go

```
@@ -2825,6 +3380,14 @@ func checkStabilityOfWPCGMUsingMetrics(fullConfig testconfig.TestConfig) {
 	checkPTPNMEAStatus(fullConfig, "1")
 }
 
+// checkStabilityOfSimGMUsingMetrics checks simulated T-GM stability
+// using only the processes and metrics available in the simulated environment.
+func checkStabilityOfSimGMUsingMetrics(fullConfig testconfig.TestConfig) {
```

golangci-lint (2.12.2)

[error] func checkStabilityOfSimGMUsingMetrics is unused

(unused)